From ba17e2d972f75a7b2ada8e17ceb90a9c43979535 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 25 Nov 2014 15:08:25 -0500
Subject: [redhat] kabi: Adding symbol kill_block_super (fs/super.c)

Message-id: <1416928105-21020-1-git-send-email-poros@redhat.com>
Patchwork-id: 100882
O-Subject: [PATCH RHEL7.1] [redhat] kabi: Adding symbol kill_block_super (fs/super.c)
Bugzilla: 1071098

Symbol already acked. This is repost, because checksum has been changed.
    Also added for ppc64le.

    Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to destroy a super block.

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/kill_block_super b/redhat/kabi/kabi-module/kabi_ppc64/kill_block_super
new file mode 100644
index 0000000..509813a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/kill_block_super
@@ -0,0 +1,2 @@
+#1-
+0x13d65753 kill_block_super vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/kill_block_super b/redhat/kabi/kabi-module/kabi_ppc64le/kill_block_super
new file mode 100644
index 0000000..d750fda
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/kill_block_super
@@ -0,0 +1,2 @@
+#1-
+0x83291510 kill_block_super vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/kill_block_super b/redhat/kabi/kabi-module/kabi_s390x/kill_block_super
new file mode 100644
index 0000000..783c591
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/kill_block_super
@@ -0,0 +1,2 @@
+#1-
+0x35edebfa kill_block_super vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/kill_block_super b/redhat/kabi/kabi-module/kabi_x86_64/kill_block_super
new file mode 100644
index 0000000..dd8e42e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/kill_block_super
@@ -0,0 +1,2 @@
+#1-
+0x73ed6e61 kill_block_super vmlinux EXPORT_SYMBOL
-- 
1.7.1